CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The change log is available [on GitHub][2].
 
 0.5.0
 =====
+
+* [#106](https://github.com/kowainik/tomland/pull/106)
```

Comment:
Could, you, please, change the link to the pull request here to the link for corresponding issue? The issue number is more important. I somehow missed it once, and now everyone specifies link to the PR...